types/ion-rangeslider/UNUSED_FILES.txt

```diff
@@ -0,0 +1,3 @@
+util/index_template.d.ts
```

Comment:
What is the purpose of adding this file to DefinitelyTyped?

Reply:
Avoids failing of tests (see comment `types/ion-rangeslider/util/parse_settings.py`)